README.md

```diff
@@ -7,3 +7,16 @@ This is a repo for studying purposes only.
 Contains a base project using OpenGL, GLEW and GLFW to build projects upon.
 
 SOIL and GLM, as well as a class for Shader compiling, are already included.
+
+## Instructions
+
+1. Open Visual Studio
+2. Click on "Open Project/Solution"
+3. Go to the folder where you downloaded this project
+4. Select "Sabertooth.sln"
+
+### If you have a Windows SDK error
```

Comment:
Please remove the "If you have". Keep title/sub-title as simple as possible.
"Windows SDK error" is just as understandable and it's easy to read.